debian/access.conf.append

```diff
@@ -15,6 +15,6 @@
 # on the machine (i.e. are in /etc/passwd).  Allow all other users to
 # login only locally.
 
--:ALL EXCEPT root admin nss-local-users:ALL EXCEPT LOCAL
+-:ALL EXCEPT root sudo admin nss-local-users:ALL EXCEPT LOCAL
```

Comment:
This actually broke the `sed 's/root admin /root /' debian/access.conf.append` in `debian/rules` on Debian. See #3.